exchanges/okx/okx_wrapper.go

```diff
@@ -0,0 +1,1188 @@
+				Tag:     response[x].Tag,
+				Chain:   response[x].Chain,
+			}, nil
+		}
+	}
+	if len(response) > 0 {
+		return &deposit.Address{
+			Address: response[0].Address,
+			Tag:     response[0].Tag,
+			Chain:   response[0].Chain,
+		}, nil
+	}
+	return nil, errDepositAddressNotFound
+}
+
+// WithdrawCryptocurrencyFunds returns a withdrawal ID when a withdrawal is submitted
+func (ok *Okx) WithdrawCryptocurrencyFunds(ctx context.Context, withdrawRequest *withdraw.Request) (*withdraw.ExchangeResponse, error) {
+	if err := withdrawRequest.Validate(); err != nil {
+		return nil, err
+	}
+	var input WithdrawalInput
+	input.ChainName = withdrawRequest.Crypto.Chain
+	input.Amount = withdrawRequest.Amount
+	input.Currency = withdrawRequest.Currency.String()
+	input.ToAddress = withdrawRequest.Crypto.Address
+	input.TransactionFee = withdrawRequest.Crypto.FeeAmount
+	input.WithdrawalDestination = "3"
+	resp, err := ok.Withdrawal(ctx, &input)
+	if err != nil {
+		return nil, err
+	}
+	return &withdraw.ExchangeResponse{
+		ID: resp.WithdrawalID,
+	}, nil
+}
+
+// WithdrawFiatFunds returns a withdrawal ID when a withdrawal is
+// submitted
+func (ok *Okx) WithdrawFiatFunds(ctx context.Context, withdrawRequest *withdraw.Request) (*withdraw.ExchangeResponse, error) {
+	return nil, common.ErrNotYetImplemented
+}
+
+// WithdrawFiatFundsToInternationalBank returns a withdrawal ID when a withdrawal is submitted
+func (ok *Okx) WithdrawFiatFundsToInternationalBank(ctx context.Context, withdrawRequest *withdraw.Request) (*withdraw.ExchangeResponse, error) {
+	return nil, common.ErrNotYetImplemented
+}
+
+// GetActiveOrders retrieves any orders that are active/open
+func (ok *Okx) GetActiveOrders(ctx context.Context, req *order.GetOrdersRequest) ([]order.Detail, error) {
+	if err := req.Validate(); err != nil {
+		return nil, err
+	}
+	if !ok.SupportsAsset(req.AssetType) {
+		return nil, fmt.Errorf("%w: %v", asset.ErrNotSupported, req.AssetType)
+	}
+	instrumentType := ok.GetInstrumentTypeFromAssetItem(req.AssetType)
+	orderType, _ := ok.OrderTypeString(req.Type)
+	requestParam := &OrderListRequestParams{
+		OrderType:      orderType,
+		After:          req.StartTime,
+		Before:         req.EndTime,
+		InstrumentType: instrumentType,
+	}
+	response, err := ok.GetOrderList(ctx, requestParam)
+	if err != nil {
+		return nil, err
+	}
+	orders := make([]order.Detail, len(response))
+	for x := range response {
+		orderSide := response[x].Side
+		var pair currency.Pair
+		pair, err = ok.GetPairFromInstrumentID(response[x].InstrumentID)
+		if err != nil {
+			return nil, err
+		}
+		for i := range req.Pairs {
+			if !req.Pairs[i].Equal(pair) {
+				continue
+			}
+			if strings.EqualFold(response[x].State, "live") {
+				response[x].State = "new"
+			}
+			var orderStatus order.Status
+			orderStatus, err = order.StringToOrderStatus(strings.ToUpper(response[x].State))
+			if err != nil {
+				log.Errorf(log.ExchangeSys, "%s %v", ok.Name, err)
+			}
```

Comment:
Add case "live" to return order.Active, then just
```suggestion
			var orderStatus order.Status
			orderStatus, err = order.StringToOrderStatus(response[x].State)
			if err != nil {
				log.Errorf(log.ExchangeSys, "%s %v", ok.Name, err)
			}
```